dashboard/app/controllers/survey_results_controller.rb

```diff
@@ -0,0 +1,28 @@
+class SurveyResultsController < ApplicationController
+  before_filter :authenticate_user!
+
+  def create
+    # puts params[:survey].to_json
+    @survey_result = SurveyResult.new(survey_result_params)
+    @survey_result.user_id = current_user.id
+
+    respond_to do |format|
+      if @survey_result.save
+        format.json { head :ok }
+      else
+        format.json { render json: @survey_result.errors, status: :unprocessable_entity }
+      end
+    end
+
+  end
+
+  # Never trust parameters from the scary internet, only allow the white list through.
+  def survey_result_params
+    valid_params = []
+    SurveyResult::ETHNICITY_ATTRS.each do |value|
+      valid_params << value.to_sym
+    end
+    params.require(:survey).permit(valid_params)
```

Comment:
if you want to get fancy:

`params.require(:survey).permit(SurveyResult::ETHNICITY_ATTRS.map(&:to_sym))`